src/vector/feature.rs

```diff
@@ -68,6 +71,11 @@ impl<'a> Feature<'a> {
                 let rv = unsafe { gdal_sys::OGR_F_GetFieldAsInteger(self.c_feature, field_id) };
                 Ok(FieldValue::IntegerValue(rv as i32))
             },
+            #[cfg(feature = "gdal_2_2")]
```

Comment:
We only need 2.0, but this is a mess anyway :-(.

Reply:
i guess we have to check if the feature is >= gdal_2_x. Maybe we could do a not "gdal_1_11"? Could you look into this?

My suggestion for the future is to rebuild the crate into feature gated modules.

Reply:
> Maybe we could do a not "gdal_1_11"? Could you look into this?

That might end up as `any(feature = "gdal_2_2", feature = "gdal_2_4", feature = "gdal_3_0")`.

> My suggestion for the future is to rebuild the crate into feature gated modules.

Sigh. I'm not sure this works too well -- it's an enum variant, after all.

Maybe we could always compile it in, but fail (or panic at runtime) if the feature wasn't enabled?

Reply:
In #86, I faced a similar issue and made a "superset" feature for the major version, maybe it would make sense to do something similar here?

```
gdal_2_2 = ["gdal_2", "gdal-sys/min_gdal_version_2_2"]
gdal_2_4 = ["gdal_2", "gdal-sys/min_gdal_version_2_4"]
gdal_2 = []
```

``` 
#[cfg(feature = "gdal_2")]
...
```